packages/perseus/src/widgets/interactive-graphs/graphs/components/angle-indicators.test.ts

```diff
@@ -94,10 +95,13 @@ describe("shouldDrawArcOutside", () => {
         ).toBe(true);
     });
 
+    // TODO: (third) Move this test to the math package
     it("should correctly calculate the angle for the given coordinates", () => {
```

Comment:
Whoops missed a TODO, I'll get this updated tomorrow.

Reply:
Question: Do you still need to move this test? :)